packages/xl-odt-exporter/src/odt/odtExporter.test.ts

```diff
@@ -0,0 +1,75 @@
+import { BlockNoteSchema, defaultBlockSpecs, PageBreak } from "@blocknote/core";
+import { testDocument } from "@shared/testDocument.js";
+import { BlobReader, TextWriter, ZipReader } from "@zip.js/zip.js";
```

Comment:
you mentioned admzip didn't work for you. Curious what was the exact case, but if zip.js works better, let's also move the other exporter(s) to zip.js so that the codebase remains consistent?

Reply:
The only other place `adm-zip` is used is in DOCX exporter tests, which run server-side. Client-side, in the browser, `adm-zip` didn't work well. Tried to patch `process` to fix it (https://github.com/cthackers/adm-zip/issues/529) but other Node-dependency-related issues popped up.

Reply:
Now using `@zip.js/zip.js` in DOCX exporter tests